languages/elixir/exercises/concept/agents/.docs/introduction.md

```diff
@@ -0,0 +1,23 @@
+The `Agent` module facilitates an abstraction for spawning processes and the _receive_, _send_ loop -- from here, we will call processes started using the `Agent` module _'agent processes'_. An _agent process_ might be chosen to represent a central shared state.
```

Comment:
> and the _receive_, _send_ loop -- from here, we will
I find this punctuation somehow off... shouldn't the words "receive" and "send" be connected somehow rather than separated by a comma? E.g. "receive-send loop"? And that's the double hyphen-minus? Maybe the part that starts with "from here..." could just be a new sentence?

```suggestion
The `Agent` module facilitates an abstraction for spawning processes and the receive-send loop. From here, we will call processes started using the `Agent` module _'agent processes'_. An _agent process_ might be chosen to represent a central shared state.
```